generic/samples/native-client/gradle.properties

```diff
@@ -0,0 +1,2 @@
+kotlin.code.style = official
+kotlinx_coroutines_version = 1.3.9-native-mt
```

Comment:
Shouldn't we have `kror_version` here as well?

Reply:
It's defined in the `gradle.properties` of the root project

Reply:
Please use native-mt-2 here